packages/mediacenter/kodi-binary-addons/inputstream.adaptive/package.mk

```diff
@@ -2,8 +2,8 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="inputstream.adaptive"
-PKG_VERSION="21.5.13-Omega"
-PKG_SHA256="2cde7954b0a73d353edc2af14306e4e42e6bf9e7f26100a54b937cb80c1309a8"
+PKG_VERSION="21.5.14-Omega"
+PKG_SHA256=""
```

Comment:
SHA is missing

Reply:
Just figured it. tools/mkpkg/update_binary-addons Is still using wget. I’ll fix that separately.

Reply:
> SHA is missing

Fixed